index.js

```diff
@@ -80,6 +80,13 @@ const path = require ('path');
 //. Indicates a JSON response. The first argument will be the response status
 //. code, and the second will be converted to JSON and sent as-is.
 //.
+//# Render :: (Number, String, Object) -> Response a
+//.
+//. Indicates a HTML response. The first argument will be the file path of the
+//. view file to render. This can be an absolute path, or a path relative to
+//. the Express views setting. The second argument indicates an object whose
+//. properties define local variables for the view.
```

Comment:
You forgot to update the documentation after adding the new parameter. How about:

*Indicates a response to be rendered using a template. The first argument will be the response status code, the second is the path to the template file, and the third is the data to inject into the template. This uses Express' `render` method under the hood, so you can configure it globally with `app.set ('view engine', engine)` and `app.set ('views', path)`.*